src/dashboard/Data/Config/ConfigConflictDiff.scss

```
@@ -0,0 +1,70 @@
+  min-width: 100%;
+  overflow: auto;
+  height: 200px;
+  min-height: 80px;
+  resize: both;
+  text-align: left;
+}
+
+.table {
+  width: 100%;
+  border-collapse: collapse;
+  font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)